cost-onprem/values.yaml

```diff
@@ -491,16 +476,19 @@ ui:
 
   oauthProxy:
     image:
-      repository: registry.redhat.io/openshift4/ose-oauth-proxy-rhel9
+      repository: quay.io/oauth2-proxy/oauth2-proxy
```

Comment:
I think if we use this one we will be in a safe territory:
https://catalog.redhat.com/en/software/containers/rhceph/oauth2-proxy-rhel9/66e9bbe69dde6f49f88ef210#overview
I'll follow up with the maintainer to see if it's exactly the one upstream one. Looks like it was updated recently (2 months ago) and it seems to be used by ceph.

Thoughts @masayag @ELK4N4 ?